README.md

```diff
@@ -11,16 +11,34 @@ There are two sets of materials available
 - [Chapters](https://github.com/openmainframeproject/cobol-programming-course/blob/master/COBOL%20Programming%20with%20VSCode.pdf) which contain the lessons, broken up into chapters.
 - [Labs](https://github.com/openmainframeproject/cobol-programming-course/tree/master/Labs) which contain the source code used in the chapters.
 
+## Build
+
+You can use [pandoc](https://pandoc.org/) to convert the markdown files provided to any other format such as DOCX, PDF, or HTML.
+
 ## Contributing
 
 Pull requests are welcome. For major changes, please open an issue first to discuss what you would like to change.
 
 All contributions must align with the [Open Mainframe Project contribution guidelines](https://github.com/openmainframeproject/tac/blob/master/process/contribution_guidelines.md), including having a DCO signoff on all commits.
 
+# Discussion
+
+You can connect with the community in a variety of ways...
+
+- [COBOL Programming Course Discussion Mailing list](https://lists.openmainframeproject.org/g/cobol-course-discussion)
+- [#cobol-programming-course channel on Open Mainframe Project Slack](https://slack.openmainframeproject.org)
+- ['COBOL technical questions' category on Open Mainframe Project Community Forums](https://community.openmainframeproject.org/c/cobol-technical-questions/16)
+
+## Providers
+
+These materials are being used by other organizations to provide COBOL training to the community. This project, nor Open Mainframe Project, doesn't endorse any one of these particular providers. If you are using these materials in your training materials, feel free [to edit and issue a pull request](https://github.com/openmainframeproject/cobol-programming-course/edit/governance-docs/README.md) to have it included.
+
+- IBM is provided a [free environment for completing the labs]().
+
 ## Governance
 
-This project is openly governed, and currently establishing its processes. Stay tuned :-)
+This project is openly governed as defined in [GOVERNANCE.md](GOVERNANCE.md).
 
 ## Credits
 
-Thank you to IBM for the initial contribution and Open Mainframe Project for hosting this project.
+The courseware materials were developed leveraging grant funds made available through American River College and proposed as a new project by IBM.
```

Comment:
@Sudharsana-Srinivasan Can you have Meredith review this so we properly acknowledge everyone?

Reply:
@jmertic: let's go with -  The training course was jointly developed by IBM, clients, American River College, and the Open Mainframe Project.